utils/utils.go

```diff
@@ -0,0 +1,126 @@
+package utils
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"io"
+	"os"
+	"strconv"
+	"net/url"
+	"reflect"
+	"bootcamp/domain/model"
+	"github.com/spf13/viper"
+	"gopkg.in/mgo.v2/bson"
+)
+
+/*
+GetEnvVar gets a given envar by key
+*/
+func GetEnvVar(key string) (string, error) {
+  viper.SetConfigFile(".env")
+  err := viper.ReadInConfig()
+  value, ok := viper.Get(key).(string)
+
+  if !ok {
+    err = errors.New("Invalid type assertion")
+  }
+
+  return value, err
+}
+
+/*
+GetObjectIdFromParams transforms an /{id} to a ObjectId
+*/
+func GetObjectIdFromParams(params map[string]string) (bson.ObjectId, error) {
```

Comment:
If the function only needs the ID then it should probably only receive an ID

Reply:
*